### PR TITLE
Added DataTable row details expanding/collapsing animation

### PR DESCRIPTION
**Pull request by cTux**
Added test for checking row details content while expanding/collapsing.
Added DataTable row details expanding/collapsing animation.
Added story.
Updated `README.md` for DataTable.

### CODE REVIEW

File: src/DataTable/DataTable.js [hunk 1]
```diff
@@ -148,9 +149,22 @@ class DataTable extends WixComponent {
 
     if (rowDetails) {
       const showDetails = !!this.state.selectedRows[rowNum];
+      const getRowRenderedDetails = (rowData, rowNum) => {
+        if (rowDetailsAnimation) {
```
Comment (Faradey27) [inline src/DataTable/DataTable.js]: what if i ll pass rowDetailsAnimation=true and showDetails=false

Comment (cTux) [reply src/DataTable/DataTable.js]: `showDetails` is rendering options for Animator, it will render just an empty Animator wrapper.

Comment (cTux) [reply src/DataTable/DataTable.js]: if we will hide `Animator` wrapper when `showDetails` will be `false` then animation will not be shown, just a component will be hidden.

**Comment by Faradey27** [comment]
i see wrong commits in the commit history

**Comment by lbelinsk** [comment]
@cTux I planned on reviewing this with @alonbt, our animation expert :)
Can you confirm that Is this branch is ready for a review?

**Comment by cTux** [comment]
@lbelinsk, not yet, please, decline this PR, i will fork this repo again.

**Comment by lbelinsk** [comment]
Closing this one, please create a new cleaner one :)

**Pull request closed without merging**